_posts/2024-02-01-wildfly-glow.adoc

```diff
@@ -0,0 +1,504 @@
+
+The XML element `<discover-provisioning-info>` enables the support for WildFly Glow (starting WildFly Maven Plugin version 5 Beta).
+[source,xml]
+----
+...
+<discover-provisioning-info/>
+...
+----
+
+The provisioning configuration is fully delegated to WildFly Glow. It will include and exclude Galleon layers according to what has been discovered 
+(exclusion of layers can be required when provisioning an HA server).
+
+==== HA server for cloud execution with support for postgresql datasource
+
+===== Configuration without WildFly Glow support
+
+[source,xml]
+----
+...
+
+<feature-packs>
+  <feature-pack>    
+    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+  </feature-pack>
+  <feature-pack>    
+    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud}</location>
+  </feature-pack>
+  <feature-pack>
+    <location>org.wildfly:wildfly-datasources-galleon-pack:${version.ds}</location>
+  </feature-pack>
+</feature-packs>
+<layers>
+    <layer>cloud-server</layer>
+    <layer>ejb</layer>
+    <layer>ejb-dist-cache</layer>
+    <layer>jpa-distributed</layer>
+    <layer>postgresql-driver</layer>
+</layers>
+<excludedLayers>
+    <layer>ejb-local-cache</layer>
+    <layer>jpa</layer>
+</excludedLayers>
+...
+----
+
+You can notice that some Galleon layers have been excluded and their HA counter parts have been included.
+
+==== Configuration with support for WildFly Glow
+
+The XML element `<discover-provisioning-info>` is evolved with the `ha` profile, `cloud` context and the `postgresql` add-on.
+
+[source,xml]
+----
+...
+<discover-provisioning-info>
+  <profile>ha</profile>
+  <context>cloud</context>
+  <add-ons>
+    <add-on>postgresql</add-on>
+  </add-ons>
+</discover-provisioning-info>
+...
+----
+
+=== And What about WildFly Bootable JAR?
+
+The https://github.com/wildfly-extras/wildfly-jar-maven-plugin[WildFly Bootable JAR Maven Plugin] has not been evolved to support Glow, 
+we have evolved the WildFly Maven Plugin to also produce WildFly Bootable JAR.
+
+So you can benefit from WildFly Glow and build WildFly executable JARs by using the same Maven plugin.
+
+To enable Bootable JAR packaging, set the `<bootable-jar>true</bootable-jar>` plugin option.
+
+Note: Details on how to produce WildFly Bootable JAR from the WildFly Maven Plugin will come in a following blog post.
+
+=== Galleon CLI vs WildFly Glow CLI
+
+==== Server with support for postgresql datasource
+
+===== Galleon CLI
+
+[source,bash]
+----
+galleon.sh install wildfly --layers=cloud-server,ejb,jsf --dir=server
+galleon.sh install org.wildfly:wildfly-datasources-galleon-pack:6.0.0.Final --layers=postgresql-datasource --dir=server
+----
+
+Note: you will then have to deploy your application into the provisioned server.
+
+===== WildFly Glow CLI
+
+[source,bash]
+----
+wildfly-glow.sh scan myapp.war --add-ons=postgresql --provision=SERVER
+----
+Note: the provisioned server contains the deployment that has been scanned.
```

Comment:
s/contains/will contain

Maybe also point out that it will be deployed under a web context root inferred from the archive name?

BTW do we have a way to make this ROOT.war from the CLI?